src/components/InputBook.js

```diff
@@ -20,8 +20,8 @@ const InputBook = () => {
   };
 
   return (
-    <div>
-      <h2>ADD NEW BOOK</h2>
+    <div className="InputContainer">
+      <h2 className="InputHeader">ADD NEW BOOK</h2>
       <form
         onSubmit={submitBookToStore}
       >
```

Comment:
- [ ] It might be a good idea to edit the width property of your ADD NEW BOOK text to match the template. ⚒️ 

**Yours**
![form](https://user-images.githubusercontent.com/79876959/145640737-0a547d45-8287-43dc-988b-278411184af7.PNG)

**Template**
![image](https://user-images.githubusercontent.com/79876959/145640801-b7378487-4e65-4423-b566-38c5ecca6f5a.png)

- [ ] Please, fix the alignment of your InputBook 

**Yours**
![space-form](https://user-images.githubusercontent.com/79876959/145641090-b31d46a7-4601-4279-b69d-3de930246387.PNG)

**Template**
![image](https://user-images.githubusercontent.com/79876959/145641146-02d86f7e-8d42-4d7a-8dea-5a2a9d096ecf.png)